docs/administration.rst

```diff
@@ -35,21 +35,23 @@ Options available to docker installations:
     ``/var/lib/docker/volumes`` on the host and you need to be root in order
     to access them.
 
-    Paperless uses 3 volumes:
+    Paperless uses 4 volumes:
 
     *   ``paperless_media``: This is where your documents are stored.
     *   ``paperless_data``: This is where auxillary data is stored. This
         folder also contains the SQLite database, if you use it.
     *   ``paperless_pgdata``: Exists only if you use PostgreSQL and contains
         the database.
+    *   ``paperless_dbdata``: Exists only if you use MariaDB and contains
```

Comment:
I don't actually find this volume anywhere?

Reply:
Fair, I was curious how that works anyway. I was basing it off of [this line](https://github.com/paperless-ngx/paperless-ngx/blob/main/docker/compose/docker-compose.portainer.yml#L43) in postgres' definition. The docker file declares it to be one name but the docs declare another, I was wondering if the docs are incorrect or something magically renames the volume.